ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+   * Fix compilation error on c++, because of a variable named new. Found and fixed by Hirotaka Niisato in #1783 
```

Comment:
Please split this into multiple lines to avoid lines of more than 80 characters. Also, please add a full stop at the end of the second sentence, and capitalize `C++`.

Reply:
@hanno-arm 
Now I update the ChangeLog.
Thank you.